packages/vuetify/vite.config.mjs

```diff
@@ -20,7 +20,7 @@ const components = files.filter(file => file.startsWith('src/labs') || !block.so
 const map = new Map(components.flatMap(file => {
   const src = readFileSync(file, { encoding: 'utf8' })
   const matches = src.matchAll(/export const (V\w+)|export { (V\w+) }/gm)
-  return Array.from(matches, m => [m[1] || m[2], file.replace('src/', '@/')])
+  return Array.from(matches, m => [m[1] || m[2], file.replace('src/', '@/').replace('.ts', '')])
```

Comment:
To assist `vue-tsc` work properly, have to remove .ts suffix for import paths, so unplugin-vue-components can yield

`VBtn: typeof import('@/components/VBtn/index')['VBtn']` 👍 
as opposed to 
`VBtn: typeof import('@/components/VBtn/index.ts')['VBtn']` 👎 

TS 4.x does not support .ts suffix in import path, and it looks like to be one of tasks in 5.x roadmap

Issue: https://github.com/microsoft/TypeScript/issues/37582
PR: https://github.com/microsoft/TypeScript/pull/51669